### ISSUE TITLE
Advice on Reducing GPU Memory Usage for MViT Training

### ISSUE THREAD

**Issue by JunweiLiang** [facebookresearch/SlowFast#496]
Hi team,

I'm currently training MViT for AVA dataset. On a 1080 TI GPU and batch_size=1, it already uses 9GB memory and would raise OOM error when trying to set batch_size=2. Does anyone has any suggestion on reducing the memory usage or is this expected? I have also tried mixed precision training but it only reduces to around 8GB, still not enough for batch_size=2. Thanks!

**Comment by JunweiLiang** [comment]
setting MODEL.ACT_CHECKPOINT to True (enable activation/gradient checkpointing with the fairscale package) could allow batch_size=8 on a 11GB GPU.
